modules/developing/pages/astream-functions.adoc

```diff
@@ -336,9 +336,9 @@ For more information, see xref:streaming-learning:functions:index.adoc[] and xre
 
 . Select input topics.
 
-. (Optional) Select output and log topics.
+. Optional: Select output and log topics.
 
-. (Optional) Configure advanced settings:
+. Optional: Configure advanced settings:
```

Comment:
For consistency with the other list items + to avoid a double ":", would this make sense?
```suggestion
. Optional: Configure advanced settings.
```